### ISSUE TITLE
Window pane a bit screwy after attempting to edit htm file

### ISSUE THREAD

**Issue by ryan4volts** [notepad-plus-plus/notepad-plus-plus#10258]
### Description of the Issue
I have used notepad++ for many years and this is the first time I have ran into an issue personally. I run default settings with only markdown plugin additionally installed. I cannot comment on the version I was running, though it was near the latest, it was from a release at least within the last 3 months. I uninstalled before looking at my previous version. After doing an uninstall and ensuring not to keep any settings, I was greeted with the below images on the latest version.

### Steps to Reproduce the Issue
I am afraid to say I don't quite know how to replicate this. Like I mentioned I simply try to edit a conventional `*.htm` signature file and was greeted with this layout.

### Actual Behavior
After opening the file I was left with the following setup:
![image](https://user-images.githubusercontent.com/33822039/126926221-c248b288-0533-405f-85df-66d2bcaf2bf7.png)

When I try to close all the panes and close/reopen the application I am greeted with the same layout.
![image](https://user-images.githubusercontent.com/33822039/126926497-1dcb0af5-c417-4fb6-9671-7778c1f3941e.png)

If I try to move any of the blank tabs, my system starts to lock up for a few seconds before notepad++ finally crashes and my machine turns back to normal.

I can confirm that while is issue is present on my login, it is not on other Users on my machine.

Windows 10.

One of the crash reports:
```
Version=1
EventType=AppHangXProcB1
EventTime=132524662319673340
ReportType=3
Consent=1
UploadTime=132524662335283078
ReportStatus=97
ReportIdentifier=dd4a67fd-0dee-4aa1-8dec-e10e54a45da5
IntegratorReportIdentifier=3adffb5d-a34e-445c-a707-fb8a303c99dc
Wow64Host=34404
Wow64Guest=332
NsAppName=notepad++.exe
OriginalFilename=Notepad++.exe
AppSessionGuid=000041a0-0001-0004-2d4c-4fc179d2d601
TargetAppId=W:0006477c1533a1c34999b4957e73a2d5737c00000904!0000fba99dc2f057e47167f78ab662b1a084ba7e6e72!notepad++.exe
TargetAppVer=2020//11//02:00:05:50!2d5919!notepad++.exe
BootId=4294967295
TargetAsId=31003
IsFatal=1
EtwNonCollectReason=1
Response.type=4
Sig[0].Name=Application Name
Sig[0].Value=notepad++.exe
Sig[1].Name=Application Version
Sig[1].Value=7.9.1.0
Sig[2].Name=Application Timestamp
Sig[2].Value=5f9f4d5e
Sig[3].Name=Hang Signature
Sig[3].Value=ff3d
Sig[4].Name=Hang Type
Sig[4].Value=134217760
Sig[5].Name=Waiting on Application Name
Sig[5].Value=CodeSetup-stable-940b5f4bb5fa47866a54529ed759d95d09ee80be.tmp
Sig[6].Name=Waiting on Application Version
Sig[6].Value=51.1052.0.0
DynamicSig[1].Name=OS Version
DynamicSig[1].Value=10.0.19041.2.0.0.256.48
DynamicSig[2].Name=Locale ID
DynamicSig[2].Value=3081
DynamicSig[22].Name=Additional Hang Signature 1
DynamicSig[22].Value=ff3ded83d62977e676152a9cbc77fd2a
DynamicSig[23].Name=Additional Hang Signature 2
DynamicSig[23].Value=455b
DynamicSig[24].Name=Additional Hang Signature 3
DynamicSig[24].Value=455b3d2722b52ea40616b8fb8b766bad
DynamicSig[25].Name=Additional Hang Signature 4
DynamicSig[25].Value=ff3d
DynamicSig[26].Name=Additional Hang Signature 5
DynamicSig[26].Value=ff3ded83d62977e676152a9cbc77fd2a
DynamicSig[27].Name=Additional Hang Signature 6
DynamicSig[27].Value=455b
DynamicSig[28].Name=Additional Hang Signature 7
DynamicSig[28].Value=455b3d2722b52ea40616b8fb8b766bad
UI[3]=Notepad++ : a free (GNU) source code editor is not responding
UI[4]=If you close the program, you might lose information.
UI[5]=Close the program
UI[6]=Close the program
UI[7]=Close the program
LoadedModule[0]=C:\Program Files (x86)\Notepad++\notepad++.exe
LoadedModule[1]=C:\WINDOWS\SYSTEM32\ntdll.dll
LoadedModule[2]=C:\WINDOWS\System32\KERNEL32.DLL
LoadedModule[3]=C:\WINDOWS\System32\KERNELBASE.dll
LoadedModule[4]=C:\WINDOWS\SYSTEM32\apphelp.dll
LoadedModule[5]=C:\WINDOWS\System32\SHLWAPI.dll
LoadedModule[6]=C:\WINDOWS\System32\msvcrt.dll
LoadedModule[7]=C:\WINDOWS\WinSxS\x86_microsoft.windows.common-controls_6595b64144ccf1df_6.0.19041.488_none_11b1e5df2ffd8627\COMCTL32.dll
LoadedModule[8]=C:\WINDOWS\System32\GDI32.dll
LoadedModule[9]=C:\WINDOWS\System32\SHELL32.dll
LoadedModule[10]=C:\WINDOWS\System32\msvcp_win.dll
LoadedModule[11]=C:\WINDOWS\System32\win32u.dll
LoadedModule[12]=C:\WINDOWS\System32\ucrtbase.dll
LoadedModule[13]=C:\WINDOWS\System32\gdi32full.dll
LoadedModule[14]=C:\WINDOWS\System32\USER32.dll
LoadedModule[15]=C:\WINDOWS\System32\CRYPT32.dll
LoadedModule[16]=C:\WINDOWS\System32\WINTRUST.dll
LoadedModule[17]=C:\WINDOWS\System32\RPCRT4.dll
LoadedModule[18]=C:\WINDOWS\System32\COMDLG32.dll
LoadedModule[19]=C:\WINDOWS\SYSTEM32\dbghelp.dll
LoadedModule[20]=C:\WINDOWS\SYSTEM32\VERSION.dll
LoadedModule[21]=C:\WINDOWS\System32\combase.dll
LoadedModule[22]=C:\WINDOWS\System32\shcore.dll
LoadedModule[23]=C:\WINDOWS\SYSTEM32\WINMM.dll
LoadedModule[24]=C:\WINDOWS\SYSTEM32\WININET.dll
LoadedModule[25]=C:\WINDOWS\System32\ADVAPI32.dll
LoadedModule[26]=C:\WINDOWS\SYSTEM32\SensApi.dll
LoadedModule[27]=C:\WINDOWS\System32\sechost.dll
LoadedModule[28]=C:\WINDOWS\System32\ole32.dll
LoadedModule[29]=C:\WINDOWS\System32\IMM32.DLL
LoadedModule[30]=C:\WINDOWS\SYSTEM32\MSASN1.dll
LoadedModule[31]=C:\WINDOWS\System32\imagehlp.dll
LoadedModule[32]=C:\WINDOWS\SYSTEM32\CRYPTSP.dll
LoadedModule[33]=C:\WINDOWS\system32\rsaenh.dll
LoadedModule[34]=C:\WINDOWS\System32\bcrypt.dll
LoadedModule[35]=C:\WINDOWS\SYSTEM32\CRYPTBASE.dll
LoadedModule[36]=C:\WINDOWS\System32\bcryptPrimitives.dll
LoadedModule[37]=C:\Program Files (x86)\Notepad++\SciLexer.dll
LoadedModule[38]=C:\WINDOWS\System32\OLEAUT32.dll
LoadedModule[39]=C:\WINDOWS\SYSTEM32\MSIMG32.dll
LoadedModule[40]=C:\WINDOWS\SYSTEM32\windows.storage.dll
LoadedModule[41]=C:\WINDOWS\SYSTEM32\Wldp.dll
LoadedModule[42]=C:\WINDOWS\SYSTEM32\uxtheme.dll
LoadedModule[43]=C:\WINDOWS\System32\MSCTF.dll
LoadedModule[44]=C:\WINDOWS\SYSTEM32\TextShaping.dll
LoadedModule[45]=C:\WINDOWS\SYSTEM32\kernel.appcore.dll
LoadedModule[46]=C:\WINDOWS\System32\clbcatq.dll
LoadedModule[47]=C:\WINDOWS\system32\dataexchange.dll
LoadedModule[48]=C:\WINDOWS\system32\dcomp.dll
LoadedModule[49]=C:\WINDOWS\system32\d3d11.dll
LoadedModule[50]=C:\WINDOWS\system32\dxgi.dll
LoadedModule[51]=C:\WINDOWS\system32\twinapi.appcore.dll
LoadedModule[52]=C:\WINDOWS\SYSTEM32\WindowsCodecs.dll
LoadedModule[53]=C:\WINDOWS\SYSTEM32\textinputframework.dll
LoadedModule[54]=C:\WINDOWS\System32\CoreUIComponents.dll
LoadedModule[55]=C:\WINDOWS\System32\CoreMessaging.dll
LoadedModule[56]=C:\WINDOWS\System32\WS2_32.dll
LoadedModule[57]=C:\WINDOWS\SYSTEM32\ntmarta.dll
LoadedModule[58]=C:\WINDOWS\SYSTEM32\wintypes.dll
LoadedModule[59]=C:\Program Files (x86)\Notepad++\plugins\DSpellCheck\DSpellCheck.dll
LoadedModule[60]=C:\Program Files (x86)\Notepad++\plugins\mimeTools\mimeTools.dll
LoadedModule[61]=C:\Program Files (x86)\Notepad++\plugins\NppConverter\NppConverter.dll
LoadedModule[62]=C:\Program Files (x86)\Notepad++\plugins\NppEditorConfig\NppEditorConfig.dll
LoadedModule[63]=C:\Program Files (x86)\Notepad++\plugins\NppExport\NppExport.dll
LoadedModule[64]=C:\WINDOWS\system32\propsys.dll
LoadedModule[65]=C:\Windows\System32\MsSpellCheckingFacility.dll
LoadedModule[66]=C:\Windows\System32\USERENV.dll
LoadedModule[67]=C:\Windows\System32\profapi.dll
LoadedModule[68]=C:\Windows\System32\Bcp47Langs.dll
LoadedModule[69]=C:\Windows\System32\OneCoreUAPCommonProxyStub.dll
OsInfo[0].Key=vermaj
OsInfo[0].Value=10
OsInfo[1].Key=vermin
OsInfo[1].Value=0
OsInfo[2].Key=verbld
OsInfo[2].Value=19041
OsInfo[3].Key=ubr
OsInfo[3].Value=685
OsInfo[4].Key=versp
OsInfo[4].Value=0
OsInfo[5].Key=arch
OsInfo[5].Value=9
OsInfo[6].Key=lcid
OsInfo[6].Value=1033
OsInfo[7].Key=geoid
OsInfo[7].Value=12
OsInfo[8].Key=sku
OsInfo[8].Value=48
OsInfo[9].Key=domain
OsInfo[9].Value=0
OsInfo[10].Key=prodsuite
OsInfo[10].Value=256
OsInfo[11].Key=ntprodtype
OsInfo[11].Value=1
OsInfo[12].Key=platid
OsInfo[12].Value=10
OsInfo[13].Key=sr
OsInfo[13].Value=0
OsInfo[14].Key=tmsi
OsInfo[14].Value=220871758
OsInfo[15].Key=osinsty
OsInfo[15].Value=1
OsInfo[16].Key=iever
OsInfo[16].Value=11.630.19041.0-11.0.220
OsInfo[17].Key=portos
OsInfo[17].Value=0
OsInfo[18].Key=ram
OsInfo[18].Value=16313
OsInfo[19].Key=svolsz
OsInfo[19].Value=237
OsInfo[20].Key=wimbt
OsInfo[20].Value=0
OsInfo[21].Key=blddt
OsInfo[21].Value=191206
OsInfo[22].Key=bldtm
OsInfo[22].Value=1406
OsInfo[23].Key=bldbrch
OsInfo[23].Value=vb_release
OsInfo[24].Key=bldchk
OsInfo[24].Value=0
OsInfo[25].Key=wpvermaj
OsInfo[25].Value=0
OsInfo[26].Key=wpvermin
OsInfo[26].Value=0
OsInfo[27].Key=wpbuildmaj
OsInfo[27].Value=0
OsInfo[28].Key=wpbuildmin
OsInfo[28].Value=0
OsInfo[29].Key=osver
OsInfo[29].Value=10.0.19041.685.amd64fre.vb_release.191206-1406
OsInfo[30].Key=buildflightid
OsInfo[30].Value=1168F1E4-7DF7-409F-A5BE-F702FDD26192.1
OsInfo[31].Key=edition
OsInfo[31].Value=Professional
OsInfo[32].Key=ring
OsInfo[33].Key=expid
OsInfo[34].Key=fconid
OsInfo[35].Key=containerid
OsInfo[36].Key=containertype
OsInfo[37].Key=edu
OsInfo[37].Value=0
FriendlyEventName=Stopped responding and was closed
ConsentKey=AppHangXProcB1
AppName=Notepad++ : a free (GNU) source code editor
AppPath=C:\Program Files (x86)\Notepad++\notepad++.exe
ReportDescription=A problem caused this program to stop interacting with Windows.
NsPartner=windows
NsGroup=windows8
ApplicationIdentity=DFC8445423127D77297A55428A69CF19
MetadataHash=-267243050
```

**Comment by ArkadiuszMichalski** [comment]
Duplicate of https://github.com/notepad-plus-plus/notepad-plus-plus/issues/9717.
